### ISSUE TITLE
The languages doesn't take aliases into account ("bash" on Linux not recognised as it's an alias in "Shell")  ( `bash` disabled or not supported.)

### ISSUE THREAD

**Issue by Speedway1** [KillianLucas/open-interpreter#867]
### Describe the bug

On linux executing commands in shell (bash) gives the error
    `bash` disabled or not supported. 

This is because the respond.py file has this loop:
                if language not in [
                    i.name.lower() for i in interpreter.computer.terminal.languages
                ] :
which doesn't take the language aliases into account. 

The aliases are defined in the file: core/computer/terminal/languages/shell.py
class Shell(SubprocessLanguage):
    file_extension = "sh"
    name = "Shell"
    aliases = ["bash", "sh", "zsh"]

### Reproduce

Install on linux
Try and run any command that uses shell, such as: "list my current directory"

### Expected behavior

Should execute the shell command

### Open Interpreter version

0.2.0

### Python version

3.10.12

### Operating System name and version

Ubuntu 22.04

### Additional context

ANYONE READING THIS AND WANTING A QUICK FIX UNTIL THE PATCH IS DONE DO THIS:

1) go to your virtual environment where the "pip install" has installed the code (e.g. open_interpreter_env)
2) edit the "respond.py" file, for example:
      vi open_interpreter_env/lib/python3.10/site-packages/interpreter/core/respond.py
3) change the code that looks like this:
                # Is this language enabled/supported?
                if language not in [
                    i.name.lower() for i in interpreter.computer.terminal.languages
                ]:
                    output = f"`{language}` disabled or not supported."
TO LOOK LIKE THIS:
                # Is this language enabled/supported?
                if language not in [
                    i.name.lower() for i in interpreter.computer.terminal.languages
                ] and language != "bash":
                    output = f"`{language}` disabled or not supported."

i.e. you are changing this line:
                ]:
to this:
                ] and language != "bash":
4) save (ZZ) and then restart "interpreter" and now your bash shell commands should work.

**Comment by cavit99** [comment]
Repeatable on OSX too.

**Comment by aj47** [comment]
I think this fixed the issue for me on windows too

**Comment by KillianLucas** [comment]
Great catch and fix @Speedway1, and many thanks for bumping it @cavit99 and @aj47. 

About to push a fix to `pip`, I'll let you know when it's up. We instead let `computer.terminal.get_languages(language)` handle retrieval instead of re-inventing retrieval outside of there, which broke it here.

**Comment by KillianLucas** [comment]
It's out! `0.2.0` should hopefully fix this issue, let me know if it doesn't:

```
pip install --upgrade open-interpreter
interpreter
> Echo hello world in bash
```

**Comment by SunnyOd** [comment]
Thanks for that, Killian! Very exciting update btw, I'm hoping to incorporate open interpreter into my project which should hopefully be out towards the end of the month 😁

Now bash's that's working, wondering which open source language models everyone will be/is using?  I've noticed that they vary a great deal in quality of response and how they're translated to commands in OI. I've tried phi-2 and a handful of models up to mixtral, I'm wondering if it's best to wait for something that's been trained specifically
